GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifiableAccountDetail.m

```diff
@@ -17,4 +17,25 @@
 #import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h"
 
 @implementation GIDVerifiableAccountDetail
+
+- (instancetype)initWithAccountDetailType:(GIDAccountDetailType)accountDetailType {
+   self = [super init];
+   if (self) {
+       _accountDetailType = accountDetailType;
+   }
+   return self;
+}
+
+// Returns dictionary mapping account detail type to scope.
++ (NSDictionary<NSNumber *, NSString *> *)scopeMapping {
+  return @{
+    @(GIDAccountDetailTypeAgeOver18) : @"https://www.googleapis.com/auth/verified.age.over18.standard"
```

Comment:
`@"https://www.googleapis.com/auth/verified.age.over18.standard"` should be a string constant.

Reply:
Done.